src/main/java/leetcode3/BalloonSolverTest.java

```diff
@@ -0,0 +1,16 @@
+package leetcode3;
+
+import org.junit.Test;
+
+import static org.junit.Assert.assertEquals;
+
+public class BalloonSolverTest {
+
+    @Test
+    public void testMaxNumberOfBalloons() {
+        assertEquals(1, BalloonSolver.maxNumberOfBalloons("nlaebolko"));
+        assertEquals(2, BalloonSolver.maxNumberOfBalloons("loonbalxballpoon"));
```

Comment:
one unit test should test one method with one set of parameters. Please divide this test